src/amo/components/LandingAddonsCard/index.js

```diff
@@ -33,9 +32,18 @@ export default class LandingAddonsCard extends React.Component<Props> {
       footerText,
       header,
       loading,
-      placeholderCount,
     } = this.props;
 
+    const filterType =
```

Comment:
This logic seems overly complicated. Would it be simpler to just pass a value for `hasThemes` into `LandingAddonsCard` as a prop? Would the caller always know if it's a `LandingAddonsCard` for themes?

Reply:
thanks for the suggestion @bobsilverberg: ) I will take another look...